src/scene/index.js

```diff
@@ -71,8 +79,8 @@ export function isHovering() {
 }
 
 export function render() {
-  const freeze = !hover || drag || !GLOBE_HOVER_FREEZE_ENABLED;
-  if (freeze) {
+  const motion = !hover || drag || !GLOBE_HOVER_FREEZE_ENABLED;
+  if (motion) {
```

Comment:
This was renamed because it did not make sense being named `freeze`...because if the value is `true`, then the globe is in `motion`, it is not frozen. 🐒